{{cookiecutter.project_slug}}/.github/workflows/test.yml

```diff
@@ -7,6 +7,17 @@ on:
       - "**"
 {% raw %}
 jobs:
+  actionlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Download actionlint
+        run: bash <(curl https://raw.githubusercontent.com/rhysd/actionlint/main/scripts/download-actionlint.bash)
+        shell: bash
+      - name: Check workflow files
+        run: ${{ steps.get_actionlint.outputs.executable }} -color
```

Comment:
The one in the root seems to be `./actionlint -color`, which one is preferred?

Reply:
Good catch. Both options are valid: https://github.com/rhysd/actionlint/blob/main/docs/usage.md#use-actionlint-on-github-actions
I aligned them :)

Reply:
Nevermind... `run: ${{ steps.get_actionlint.outputs.executable }} -color` fails as the batch file is not generating any output.